docs/pages_en/advanced/helm/deploy_process/resource_order.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Deployment order
+permalink: advanced/helm/deploy_process/deployment_order.html
+---
+
+By default, all resources have [`werf.io/weight: 0`]({{ "/reference/deploy_annotations.html#resource-weight" | true_relative_url }}), thus will be applied and tracked concurrently. If you want to specify the order in which resources will be applied and tracked you can specify different weights for your resources.
+
+Before deployment phase resources are grouped based on their weight, then the group with the lowest associated weight is applied to the cluster and tracked until resources in this group are ready. After all resources in group are successfully deployed, the deployment of the resources with the next lowest weight is started. This continues until all release resources are deployed.
+
+> "werf.io/weight" will work only for non-Hook resources. For Hooks use "helm.sh/hook-weight".
```

Comment:
```suggestion
All resources are applied and tracked concurrently, since they all have the same weight by default ([`werf.io/weight: 0`]({{ "/reference/deploy_annotations.html#resource-weight" | true_relative_url }})). To specify the order in which resources will be applied and tracked, you can set different weights for them.

Before the deployment phase, werf groups the resources based on their weight, then applies the group with the lowest associated weight and waits until the resources from that group are ready. After all the resources from that group have been successfully deployed, werf proceeds to deploy the resources from the group with the next lowest weight. The process continues until all release resources have been deployed.

> Note that "werf.io/weight" works only for non-Hook resources. For Hooks, use "helm.sh/hook-weight".
```

Reply:
> All resources are applied and tracked concurrently, since they all have the same weight by default.
I think we should emphasize that this is only the case with default weights on all resources. Now it looks like any resources applied concurrently not by default, but because they have weight 0 (for whatever reason).

Reply:
Rephrased:
By default, resources are applied and tracked simultaneously because they all have the same initial ([`werf.io/weight: 0`]({{ "/reference/deploy_annotations.html#resource-weight" | true_relative_url }})) (which means that you haven't changed it). However, you can change the order in which resources are applied and tracked by setting different weights for them.